2025.csv

```diff
@@ -33,8 +33,9 @@ PyCon India,2025-09-12,2025-09-15,"Bengaluru, India",IND,NIMHANS Convention Cent
 PyCon UK,2025-09-19,2025-09-22,"Manchester, United Kingdom",GBR,CONTACT,2025-05-11,2025-05-11,https://2025.pyconuk.org/,https://2025.pyconuk.org/call-for-proposals/,https://2025.pyconuk.org/sponsorship/
 PyCon JP,2025-09-26,2025-09-27,"Hiroshima, Japan",JPN,International Conference Center Hiroshima,,,https://2025.pycon.jp/,,
 PyCon Estonia,2025-10-02,2025-10-03,"Tallinn, Estonia",EST,,,2025-01-15,https://pycon.ee,https://docs.google.com/forms/d/e/1FAIpQLScn-H2turgLHVbbm6g3t6MeOBmStlgQwEay8J1qfypfERKw0A/viewform,https://pycon.ee/#faq
-Plone Conference & PyCon Finland,2025-10-13,2025-10-19,"Jyväskylä, Finland",FIN,University of Jyväskylä,,2025-07-31,https://2025.ploneconf.org,https://2025.ploneconf.org/call-for-papers,https://2025.ploneconf.org/sponsors
+Plone Conference,2025-10-13,2025-10-19,"Jyväskylä, Finland",FIN,University of Jyväskylä,,2025-07-31,https://2025.ploneconf.org,https://2025.ploneconf.org/call-for-papers,https://2025.ploneconf.org/sponsors
 Swiss Python Summit,2025-10-16,2025-10-17,"Rapperswil, Switzerland",CHE,OST Campus,,2025-05-31,https://www.python-summit.ch,https://www.python-summit.ch/cfp/,https://www.python-summit.ch/sponsoring/
+PyCon Finland,2025-10-17,2025-10-17,"Jyväskylä, Finland",FIN,University of Jyväskylä,,2025-07-31,https://pyconfi.ploneconf.org,https://2025.ploneconf.org/call-for-papers,https://2025.ploneconf.org/sponsors
```

Comment:
@hugovk FYI, Firefox is complaining about https://pyconfi.ploneconf.org having an invalid TLS certificate.
It's both expired in 2019 and has `CN=2015.ploneconf.org`. If you know the organizers, they should be notified about their web server misconfiguration.

Reply:
I just realized it also serves a Plone Conf 2015 website.

Reply:
Thanks, I actually meant to change it to https://fi.pycon.org which is redirecting properly in Firefox.

```suggestion
PyCon Finland,2025-10-17,2025-10-17,"Jyväskylä, Finland",FIN,University of Jyväskylä,,2025-07-31,https://fi.pycon.org,https://2025.ploneconf.org/call-for-papers,https://2025.ploneconf.org/sponsors
```

We don't need to use https://pyconfi.ploneconf.org/ any more, that was temporary whilst waiting for the above to be set up.